docs/run_test/quick_run.md

```diff
@@ -79,10 +53,10 @@ against Linux VM on Azure.
     }
     ```
 
-    AzureCloud in this example is set to default. You do not need to do that as
-    long as you provide the correct Azure subscription ID.
+    Note AzureCloud in this example is set to default, but you do not need to do
```

Comment:
This sentence is confusing. It needs to explain, the subscription, which is taken, will be used to run tests. So user needs to make sure it's a subscription for testing, not use subscription for production or other purposes.

"AzureCloud in this example is set to default."

Reply:
I agree and will add clarification for distinguishing between subscription for testing and subscription for production. But I don't see where the confuse is. Did you mean just removing the initial "note"?

Reply:
AzureCloud is a random name of a subscription, the name isn't meaningful. This sentence sounds like it will set a subscription, which the name is AzureCloud, to default subscription. But we don't even know what subscriptions user have.

Reply:
All right, I get it now. My point was to clarify the `isDefault=True` attribute has nothing to do with our preparation.